src/main/java/picard/sam/AbstractAlignmentMerger.java

```diff
@@ -86,6 +87,7 @@
     public static final int MAX_RECORDS_IN_RAM = 500000;
 
     private static final char[] RESERVED_ATTRIBUTE_STARTS = {'X', 'Y', 'Z'};
+    private static final int MAX_CROSS_SPECIES_CONTAM_WARN = 1_000;
```

Comment:
Also, is there really value in even 1,000 warnings?  I would think that after about 20-100 I'd want them to stop cluttering up my log file.